src/sklearn_evaluation/plot/regression.py

```diff
@@ -0,0 +1,75 @@
+"""
+Plotting functions for regression plots.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from sklearn_evaluation.telemetry import SKLearnEvaluationLogger
+from sklearn.linear_model import LinearRegression
+
+
+@SKLearnEvaluationLogger.log(feature='plot')
+def residual(y_true, y_pred, ax=None):
+
+    _check_parameter_validity(y_true, y_pred)
+
+    if ax is None:
+        ax = plt.gca()
+
+    # horizontal line for residual=0
+    ax.axhline(y=0)
+
+    ax.scatter(y_pred, y_true-y_pred)
+
+    _set_ax_settings(ax, 'Predicted Value', 'Residuals')
+    return ax
+
+@SKLearnEvaluationLogger.log(feature='plot')
+def prediction_error(y_true, y_pred, ax=None):
+    _check_parameter_validity(y_true, y_pred)
+    if ax is None:
+        ax = plt.gca()
+
+    # x = y_true.reshape(-1,1);
+    # best fit line
+    model = LinearRegression()
+    model.fit(y_true.reshape((-1, 1)), y_pred)
```

Comment:
why are we fitting a model here? we should leave that to the user and only get the input data. they might want to use another package

Reply:
Oh this was the best fit line between the measured values and predicted values, so linear regression shows the difference between the error and the identity line (when measured values == predicted values)